packages/types/src/span.ts

```diff
@@ -23,7 +23,12 @@ export type SpanAttributeValue =
   | Array<null | undefined | number>
   | Array<null | undefined | boolean>;
 
-export type SpanAttributes = Record<string, SpanAttributeValue | undefined>;
+export type SpanAttributes = {
+  'sentry.origin': string;
+  'sentry.op': string;
+  'sentry.source': string;
+  'sentry.sample_rate': number;
+} & Record<string, SpanAttributeValue | undefined>;
```

Comment:
I'm not sure how important this is but I'm a bit concerned that the  `SpanAttributes` type doesn't have any pre-set keys. Should we do this? I think it makes auto completion easier when setting attributes but it could also be a mistake. can't use the `SENTRY_SEMANTIC_ATTRIBUTE` strings as const here because they're set in core.

Reply:
Also, this is technically out of scope of this PR so happy to extract it to its own

Reply:
nevermind, I'm gonna extract this to it's own PR